hollow-jar/full/pom.xml

```diff
@@ -0,0 +1,146 @@
+                                    <groupId>org.wildfly.swarm.servers</groupId>
+                                    <artifactId>full</artifactId>
+                                    <version>${version.org.wildfly.swarm}</version>
+                                    <classifier>hollowswarm</classifier>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>net.java.xadisk</groupId>
+                                    <artifactId>xadisk</artifactId>
+                                    <version>${version.net.java.xadisk}</version>
+                                    <type>rar</type>
+                                </artifactItem>
+                            </artifactItems>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>org.honton.chas</groupId>
+                <artifactId>process-exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>start</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                          <goal>start</goal>
+                        </goals>
+                        <configuration>
+                            <name>swarm</name>
+                            <workingDir>${project.build.directory}</workingDir>
+                            <arguments>
+                                <argument>${java.home}/bin/java</argument>
+                                <argument>-jar</argument>
+                                <argument>dependency/full-hollowswarm.jar</argument>
+                                <argument>${project.build.finalName}.${project.packaging}</argument>
+                                <argument>-s</argument>
+                                <argument>classes/project-defaults.yml</argument>
+                            </arguments>
+                            <healthCheckUrl>http://localhost:8080/health</healthCheckUrl>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>start-resource-adapter</id>
+                        <phase>pre-integration-test</phase>
+                        <goals>
+                          <goal>start</goal>
+                        </goals>
+                        <configuration>
+                            <name>swarm</name>
+                            <workingDir>${project.build.directory}</workingDir>
+                            <arguments>
+                                <argument>${java.home}/bin/java</argument>
+                                <argument>-jar</argument>
+                                <argument>dependency/full-hollowswarm.jar</argument>
+                                <argument>dependency/xadisk.rar</argument>
```

Comment:
I'd like to have _one_ execution for everything. If we can't easily deploy the RAR with a hollow JAR, then that's a bug.

Reply:
We can, but we need to provide as a deployment (instead of a WAR), right?

Reply:
Well we _need to_ be able to deploy a RAR together with an application. Just like with regular fat JAR.